notebook/static/notebook/js/menubar.js

```diff
@@ -189,6 +189,27 @@ define([
             that._nbconvert('html', false);
         });
 
+        this.element.find('#document_details').click(function () {
+            var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
+            var table = $('<table></table>');
+            var body = $('<tbody></tbody>');
+            
+            var lmod = that.notebook.last_modified;
+            var last_modified_str = months[lmod.getMonth()] + ' ' +
+                lmod.getDate() + ', ' + lmod.getFullYear() + ' ' + lmod.getHours() + ':'+ lmod.getMinutes();
+            
+            body.append('<tr><td><b>File: </b></td><td>' + that.notebook.notebook_name + '</td></tr>');
```

Comment:
We shouldn't construct HTML like this, because the name and the path might contain characters that are significant in HTML. We should construct the relevant HTML node and then call `.text(that.notebook.notebook_name)` on it to set the text safely.